src/rocksdb_options.rs

```diff
@@ -127,6 +127,22 @@ impl BlockBasedOptions {
         }
     }
 
+    pub fn set_ribbon_filter(&mut self, bits_per_key: f64, bloom_before_level: i32) {
```

Comment:
if $opt.use_bloom_filter {
            /*block_base_opts.set_bloom_filter(
                $opt.bloom_filter_bits_per_key as f64,
                $opt.block_based_bloom_filter,
            );*/
            block_base_opts.set_ribbon_filter($opt.bloom_filter_bits_per_key as f64, -1);
            block_base_opts.set_optimize_filters_for_memory(true);
            block_base_opts.set_whole_key_filtering($opt.whole_key_filtering);
        }
And tikv config:
 rocksdb.defaultcf.format-version: 5
    rocksdb.lockcf.format-version: 5
    rocksdb.writecf.format-version: 5

I updated the code like above, but the new SSTs are still using bloom filters. 

Table Properties:
# data blocks=1028
# entries=122319
# deletions=0
# merge operands=0
# range deletions=0
raw key size=5382036
raw average key size=44.000000
raw value size=30055495
raw average value size=245.714035
data block size=33573477
index block size (user-key? 1, delta-value? 1)=35933
filter block size=107525
# entries for filter=122320
(estimated) table size=33716935
filter policy name=rocksdb.BuiltinBloomFilter
prefix extractor name=FixedSuffixSliceTransform
column family ID=2
column family name=write
comparator name=leveldb.BytewiseComparator
merge operator name=nullptr
property collectors names=[tikv.mvcc-properties-collector,tikv.range-properties-collector]
SST file compression algo=NoCompression
SST file compression options=window_bits=-14; level=32767; strategy=0; max_dict_bytes=0; zstd_max_train_bytes=0; enabled=0; max_dict_buffer_bytes=0; 
creation time=1680634784
time stamp of earliest key=1680634784
file creation time=1680634870
slow compression estimated data size=0
fast compression estimated data size=0
DB identity=d71c6f83-1907-41bb-9098-49edf385c7fc
DB session identity=HV8AHENLLNPXO27W25IG
DB host id=ip-172-31-8-192.us-west-2.compute.internal
original file number=563317

Reply:
Added a name suffix, it should be displayed as `rocksdb.BuiltinBloomFilter.Ribbon` now.